Doc/Manual/Go.html

```diff
@@ -60,38 +60,44 @@
 
 
 <p>
-Go is a compiled language, not a scripting language.  However, it does
-not support direct calling of functions written in C/C++.  The cgo
-program may be used to generate wrappers to call C code from Go, but
-there is no convenient way to call C++ code.  SWIG fills this gap.
+Go does not support direct calling of functions written in C/C++.  The
+<a href="https://golang.org/cmd/cgo/">cgo program</a> may be used to generate
+wrappers to call C code from Go, but there is no convenient way to call C++
+code.  SWIG fills this gap.
 </p>
 
 <p>
-There are (at least) two different Go compilers.  One is the gc
-compiler, normally invoked via the go tool.  The other
-is the gccgo compiler, which is a frontend to the gcc compiler suite.
-The interface to C/C++ code is completely different for the two Go
-compilers.  SWIG supports both, selected by a command line option.
+There are (at least) two different Go compilers.  The first is the Go compiler
+of the <a href="https://golang.org/doc/install">Go distribution</a>.
+Since Go 1.5 the Go compiler is part of the <a href="https://golang.org/cmd/go/">
```

Comment:
The issue for me is that the 'gc' tool/binary is gone with Go 1.5.  There is no more 'go tool gc' and the link https://golang.org/cmd/gc/ is dead.  'gc' itself still exists but it is now part of the internal package under 'cmd/compile/internal'.  So from an user perspective 'gc' is gone and IMHO we should no longer talk about 'gc' if it is something that is no longer usable by users.  ;-)

Anyhow... What would you write?  How about "One is the Go compiler of the Go distribution." and leave the "Since Go 1.5 ..." part out?  I personally would like to give a link with additional information and so a link to 'go build' seems to be most appropriate to me.  What do you think?

Should I update the Examples/go/index.html document as well as it still mentions the gc compiler?
https://htmlpreview.github.io/?https://github.com/swig/swig/blob/master/Examples/go/index.html